bundles/k3d-slim-dev/uds-bundle.yaml

```diff
@@ -30,6 +30,9 @@ packages:
             - name: policies
               description: "Minio policies"
               path: policies
+            - name: K3D_EXTRA_ARGS
+              description: "Extra arguments for k3d cluster creation"
+              path: k3d_extra_args
```

Comment:
I don't think this is correct/necessary? Where it's placed is under the `minio` chart so this wouldn't actually flow where you expect. Since `k3d_extra_args` is already exposed as a zarf var, we shouldn't need any override in the bundle itself.